src/Microsoft.DotNet.Build.Tasks.Feed/PackageFiles/Microsoft.DotNet.Build.Tasks.Feed.targets

```diff
@@ -63,7 +63,11 @@
                     AccountKey="$(AccountKey)"
                     ItemsToPush="@(_ItemsToPush)"
                     Overwrite="$(PushToBlobFeed_Overwrite)"
-                    MaxClients="$(PushToBlobFeed_MaxClients)" />
```

Comment:
please document them above in the comment.

Reply:
Documenting it made me notice that `ManifestName` mismatched `GitHubRepositoryName`, so I changed it to make the behavior effectively the same but allow specifying `ManifestName` directly.